docs/2_2_common_mechanisms.adoc

```diff
@@ -704,8 +704,8 @@ is a time-based Clock whose next interval is unchangeably known right after the
 +
 The next Clock tick at time instant latexmath:[t_i] is defined as: +
 latexmath:[\begin{align*}
-\mathbf{t}_0 &:= \mathbf{t}_{\mathit{start}} + \mathbf{T}_{interval, 0} \\
-\mathbf{t}_i &:= \mathbf{t}_{i-1} + \mathbf{T}_{interval, i} \qquad i = 1,2,3,{...}
+t_0 &:= t_{\mathit{start}} + \mathbf{T}_{interval, 0} \\
```

Comment:
\mathbf{T}_{interval, 0}?
Several occurances

Reply:
Unboldet all T, please unresolve if you find one missing.